<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrainerGui</class>
 <widget class="QMainWindow" name="TrainerGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1351</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TrainerGui</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="img1Button">
        <property name="text">
         <string>Select Image 1...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="img2Button">
        <property name="text">
         <string>Select Image 2...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="trainingFileButton">
        <property name="text">
         <string>Select Training File...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="trainingGroup">
        <property name="title">
         <string>Training Example Type</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QRadioButton" name="positiveRadio">
             <property name="text">
              <string>Positive</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="negativeRadio">
             <property name="text">
              <string>Negative</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="ClickLabel" name="img1Label" native="true">
        <property name="minimumSize">
         <size>
          <width>201</width>
          <height>0</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>CrossCursor</cursorShape>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ClickLabel" name="img2Label" native="true">
        <property name="minimumSize">
         <size>
          <width>201</width>
          <height>0</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>CrossCursor</cursorShape>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ClickLabel</class>
   <extends>QWidget</extends>
   <header>clicklabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
